### DISCUSSION TITLE
export_formats

### DISCUSSION THREAD

**Discussion by firasendoubi123** [ultralytics/yolov5#11776] [General]
I can't resolve this error : Exception: cannot import name 'export_formats' from 'export'

**Comment by Lucky-BenXie** [comment]
> I can't resolve this error : Exception: cannot import name 'export_formats' from 'export'
I encountered the same problem too

**Comment by moronya** [comment]
I am getting the same error too. Did anyone get a workaround the issue?
I tried **pip install export**
However, the error is still persisting.
